eng/common/build.sh

```diff
@@ -130,6 +130,9 @@ while [[ $# > 0 ]]; do
     /p:*)
       properties="$properties $1"
       ;;
+    /m:*)
```

Comment:
This change is only intermediate to unblock Unix perf runs and needs to be done in arcade.

Reply:
https://github.com/dotnet/arcade/pull/1694